lib/Listeners/RegisterTemplateCreatorListener.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Morris Jobke <hey@morrisjobke.de>
+ *
+ * @author Morris Jobke <hey@morrisjobke.de>
```

Comment:
Probably outdated, right?

We might also start using the SPDX headers right away when adding new files.

Reply:
Adjusted the copyright, but would rather keep moving to SPDX separate and do it with the migration script from Joas then for all files